### PR TITLE
257 feature jmeter test duration instead of loop count

### LINKED ISSUE #257: [FEATURE] jmeter should use test duration as opposed to loop count
**Is your feature request related to a problem? Please describe.**
Currently, jmeter tests use the loop count to indicate how many times a test plan should run.
This means that when there is a performance regression, the test also runs longer.

**Describe the solution you'd like**
Leverage Test Plan duration to set how long a test should run (e.g. 5mins)
Locust is already leveraging this behaviour.

### PR DESCRIPTION
**Pull request by eemrdog**

### CODE REVIEW

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx]: Please remove jmeter/simplenodeservice_load.jmx if not used anymore. Thanks!

**Review by tobigremmer-dt — CHANGES_REQUESTED** [review]
Please check whether jmeter/simplenodeservice_load.jmx is still in use, delete otherwise.

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx (hunk 1 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx]: simplenodeservice_perfcheck.jmx is also not used by any curated roles. But may an external use case utilize them in their project?

Comment (tobigremmer-dt) [reply user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx]: Probably not... They're all obsolete. If so, they can be added to the external use case. Please clean them up. Thanks!

Comment (eemrdog) [reply user-skel/ansible/roles/app-simplenode/files/repos/simplenode/jmeter/simplenodeservice_test_by_duration.jmx]: ok deleted them

**Review by tobigremmer-dt — APPROVED** [review]
Looks good to me!

**Pull request merged**